packages/contentful--app-scripts/src/activate/index.ts

```diff
@@ -0,0 +1,37 @@
+import { activateBundle } from './activate-bundle';
+import { getActivateSettingsArgs } from './get-activate-args';
+import { buildBundleActivateSettings } from './build-bundle-activate-settings';
+import { Organization } from '../organization-api';
+import { Definition } from '../definition-api';
+
+export interface ActivateOptions {
+  bundleId: string;
+  organizationId?: string;
+  definitionId?: string;
+  token?: string;
+  host?: string;
+}
+
+export interface ActivateSettings {
+  bundleId: string;
+  organization: Organization;
+  definition: Definition;
+  accessToken: string;
+  host?: string;
+}
+
+const interactive = async (options: ActivateOptions) => {
+  const settings = await buildBundleActivateSettings(options);
+  await activateBundle(settings);
+};
+
+const nonInteractive = async (options: ActivateOptions) => {
+  const settings = await getActivateSettingsArgs(options);
+
+  await activateBundle(settings);
+};
+
+export default {
```

Comment:
Can we do named exports here? Or would that be a breaking change?

Reply:
Should work.